estate/models/estate_property.py

```diff
@@ -0,0 +1,22 @@
+from odoo import models,fields
+
+class EstateProperty(models.Model):
+    _name= 'estate.property'
+    _description= 'This is a estate property app'
+
+    def set_selection(self):
+        return 'north'
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
```

Comment:
```suggestion
    is_garage = fields.Boolean()
    is_garden = fields.Boolean()
```
TRy to define boolean fields like this which will make it more easier to understand and code.